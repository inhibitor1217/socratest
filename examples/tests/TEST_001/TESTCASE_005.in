3.1415926535
double
1
int
