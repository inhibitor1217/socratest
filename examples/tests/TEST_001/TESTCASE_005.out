3.1
string
